altair/sphinxext/altairgallery.py

```diff
@@ -271,8 +271,8 @@ def main(app):
             "Histograms": [],
             "Maps": [],
             "Interactive Charts": [],
+            "Advanced Calculations": [],
             "Case Studies": [],
-            "Other Charts": [],
         }
```

Comment:
```suggestion
            "Other Charts": [],
        }
```

The docbuild is failing without this and we still have some logic to assign uncategorized charts as "Other Charts", so let's keep it here as a backup.

Reply:
I am still running into another issue locally with `multiple_interactions.py` which raises `Error: Unrecognized signal name: "Big_Budget_Films_tuple_fields"` after the suggested. But let's commit this suggestion first and see if it fixes the CI since you haven't modified the `multiple_interactions.py` file in this PR.